os/win/window.cpp

```diff
@@ -263,6 +264,11 @@ WindowWin::WindowWin(const WindowSpec& spec)
 
   SetWindowLongPtr(m_hwnd, GWLP_USERDATA, reinterpret_cast<LONG_PTR>(this));
 
+  if (winApi.GetWindowDpiAwarenessContext) {
+    double scaleFactor = winApi.GetDpiForWindow(m_hwnd) / (double) USER_DEFAULT_SCREEN_DPI;
+    m_scale = std::max(1, (int)(m_baseScale * scaleFactor + 0.51));
```

Comment:
I think we could use the ceil funciton instead:
```suggestion
    m_scale = std::ceil(m_baseScale * scaleFactor);
```